src/components/forms/LoginForm.jsx

```diff
@@ -0,0 +1,45 @@
+
+import { useForm } from 'react-hook-form';
+
+export default function LoginForm({ onSubmit }) {
```

Comment:
Is the login going to have a password confirmation or is it the signup? or is it like reusable for both?

Reply:
I suggest to do reusable form for both